src/main/webapp/resources/js/pages/analysis/components/AnalysisError.jsx

```diff
@@ -1,12 +1,140 @@
-import React from "react";
-import { Typography } from "antd";
+/*
+ * This file renders the error details for the analysis
+ */
 
-const { Title } = Typography;
+/*
+ * The following import statements makes available all the elements
+ * required by the component encompassed within
+ */
+
+import React, { Suspense, useContext, useEffect, useState } from "react";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { Alert, Spin, Tabs } from "antd";
+import { getJobErrors } from "../../../apis/analysis/analysis";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import styled from "styled-components";
+
+const GalaxyJobInfoTab = React.lazy(() => import("./GalaxyJobInfoTab"));
+const GalaxyParametersTab = React.lazy(() => import("./GalaxyParametersTab"));
+const StandardErrorTab = React.lazy(() => import("./StandardErrorTab"));
+const StandardOutputTab = React.lazy(() => import("./StandardOutputTab"));
+
+const TabPane = Tabs.TabPane;
+
+export default function AnalysisError(props) {
+  const { analysisContext } = useContext(AnalysisContext);
+  const [jobErrors, setJobErrors] = useState(null);
+  const [currActiveKey, setCurrActiveKey] = useState(1);
+
+  const StyledTabs = styled(Tabs)`
+    .ant-tabs-tab {
+      @media only screen and (min-width: 800px) {
+        width: 200px;
+      }
+    }
+  `;
+
+  // Sets the job errors into a local state variable on page load
+  useEffect(() => {
+    getJobErrors(analysisContext.analysis.identifier).then(data => {
+      setJobErrors(data);
+    });
+  }, []);
+
+  // Sets the current activekey for the 'Pass' tabs
+  function updateActiveKey(key) {
+    setCurrActiveKey(key.charAt(key.length - 1));
+  }
 
-export default function AnalysisError() {
   return (
-    <Title level={2} className="t-page-title">
-      Job Error
-    </Title>
+    <>
```

Comment:
Try to break this section into components, you also don't need the `<>`

Reply:
Already done. However removing the <> breaks the page and there are a bunch of errors

Reply:
Fixed in 6e887213d951c73e40a422689baebcb640ab1be4